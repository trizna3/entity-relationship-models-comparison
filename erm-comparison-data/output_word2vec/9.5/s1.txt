assembly-assemblies;deputies
documents-documents
vote options-EMPTY
persons-EMPTY
agenda-items
agenda type-EMPTY
GENERALIZATION_TO_11_ASSOCIATION:S-item types-items
CHANGE_CARDINALITY:S-assemblies-deputies
CHANGE_CARDINALITY:S-deputies-assemblies
CONTRACT_11_ASSOCIATION:S-assemblies-deputies
