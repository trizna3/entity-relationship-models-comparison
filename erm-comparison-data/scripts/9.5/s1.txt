entity set nrsr {
};

entity set deputies {
	"name"
	"lastname"
};

entity set assemblies {
	"name"
	"duration"
};

entity set items {
	"name"
};

entity set "vote history" {
	"when"
	"how"
};

entity set "item types" {
	"minimal count to pass"
};

entity set documents {
	"name"
};

entity set "vote types" {
	"for"
	"against"
	"pass"
	"absent"
};

association between 
{
	nrsr ONE
	deputies MANY
}{};

association "participate in" between 
{
	assemblies MANY
	deputies MANY
}{};

association between 
{
	items MANY
	documents MANY
}{};

association between 
{
	"vote history" ONE
	deputies MANY
}{};

association between 
{
	assemblies ONE
	items MANY
}{};

association between 
{
	deputies ONE
	items MANY
	assemblies MANY
}{};

association between 
{
	"vote types" ONE
	deputies MANY
}{};

generalization from "item types" to items;