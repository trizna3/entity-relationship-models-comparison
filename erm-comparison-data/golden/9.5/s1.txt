persons-deputies
assembly-assemblies
agenda-items
documents-documents
agenda type-item types
vote options-EMPTY